### PR TITLE
Add a nascent version of the ddit command line tool.

### PR DESCRIPTION
**Pull request by mschaef-da**
Merging in advance of splitting out into `daml-dit-ddit`.

### CODE REVIEW

File: daml_dit_api/ddit/main.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    setup_argparse_show(subparsers.add_parser('show', parents=[parent]))
+
+    kwargs = vars(parser.parse_args())
+    subcommand_name = kwargs.pop('subcommand_name')
+
+    verbose = kwargs.pop('verbose')
+
+    setup_default_logging(level=logging.DEBUG if verbose else logging.INFO)
+
+    globals()[f'subcommand_{subcommand_name}'](**kwargs)
```
Comment (da-tanabe) [inline daml_dit_api/ddit/main.py:39]: hahahaha I can't decide whether I love it or hate it… :-)

Comment (mschaef-da) [reply daml_dit_api/ddit/main.py:39]: Personally, I hate it and am looking forwards to replacing it later this week.

Comment (mschaef-da) [reply daml_dit_api/ddit/main.py:39]: This whole "python symbols as raw strings" thing gives me an idea for a spooky Halloween costume.

Comment (da-tanabe) [reply daml_dit_api/ddit/main.py:39]: Actually…this is probably better as an explicit interface.

Note that you can also use https://docs.python.org/3/library/typing.html#typing.Protocol as a way of defining an informal spec that doesn't require actual inheritance (`Protocol` basically formalizes duck typing in Python).

Comment (mschaef-da) [reply daml_dit_api/ddit/main.py:39]: Let me do this once it's in its own repo.

**Review by da-tanabe — APPROVED** [review]

**Pull request merged**